src/web/theme/pages/OurSelection/SiteCard/Sitecard.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import PropType from "prop-types";
+import "./SiteCard.scss";
+
+const SiteCard = ({ site }) => {
+  return (
+    <div className="site-card">
+      <div className="site-card__image">
+        <img src={site.image} />
+      </div>
+      <div className="site-card__content">
+        <div className="site-card__content__name">
+          <span>{site.name}</span>
```

Comment:
Do you really need the span here ? The div should be enough

Reply:
I use an after css property ... the span here is to make a margin between the site-card__content__name children.